AirCasting/Dashboard/SessionListView.swift

```diff
@@ -44,6 +46,10 @@ struct SessionsListView: View {
                 if selectedSection.allowsRefreshing {
                     RefreshControl(coordinateSpace: .named(listCoordinateSpaceName), isRefreshing: $isRefreshing)
                 }
+                if selectedSection == .following && isDownloading {
```

Comment:
maybe we can do the same as with refresh control - move the `selectedSection == .following` to a computed var like `selectedSection.shouldShowMeasurementDownloadProgress` or smth like that? This way we'll have all of those conditions in a single place